types/ioredis/ioredis-tests.ts

```diff
@@ -80,23 +80,16 @@ new Redis({
 });
 
 const pub = new Redis();
-redis.subscribe('news', 'music', (err: any, count: any) => {
```

Comment:
Why this removed?

Reply:
Right I meant to ask about this, the callback technically comes after the vararg which is not possible to model in TS (yet) so I removed this test as I couldn't satisfy it.
The possible options here are to add multiple overloads and an `array` variant instead eg.

```ts
subscribe(sub1: string, cb: (err: Error, count: number) => void): Promise<number>;
...
subscribe(subs: string[], cb: (err: Error, count: number) => void): Promise<number>;
```

Since ioredis flattens all arguments by default, this would be a valid (and usable) definition.

Reply:
@plantain-00 ping